scripts/rhoso_adoc_docs_to_text.py

```
@@ -1364,8 +1366,8 @@ def __exit__(self, exc_type, exc_val, exc_tb):
                 # Try to remove the lock file (best effort)
                 try:
                     self.lock_path.unlink()
-                except Exception:
-                    pass
+                except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)